### ISSUE TITLE
Resizable Visor

### ISSUE THREAD

**Issue by jhaberstro** [binaryage/totalterminal#23]
It is unwieldy to have to change the column and row sizes in the Terminal's inspector. Being able to drag the edges to resize, and having default sizes would be great.

**Comment by pehrlich** [comment]
I would love this as well.  As a snow leopard plugin, I was able to configure the initial size, which was really nice, but I haven't been able to find that configuration as an app.

**Comment by andyras** [comment]
I agree heartily with this feature request, though I would like to understand why this would be a difficult feature to implement.

For my kind of work, which involves running scripts in the terminal which produce/update a plot or image on another part of the screen, I frequently change the size/width of the terminal window to accomodate other elements.

I love using TotalTerminal as my primary terminal, but it is my frequent wish that I could adjust the size of the terminal window (e.g. in left-stretch or top-stretch configuration) on the fly.

I am guessing (perhaps wrongly--I don't know how Mac OS UI elements work) that one function of the TotalTerminal plugin is to disable the size-adjustment arrow that appears when mousing over the edge of the terminal window.  If so, adding this feature would hopefully be a simple matter of adding the option not to remove that adjustment arrow on the free edge of the terminal window.

**Comment by darwin** [comment]
I would like this feature too. If you implement it, I will merge it :)

btw. CMD+OPT+F gives you temporary full-screen dimensions.

**Comment by darwin** [comment]
One more note. I don't explicitly disable the adjustment arrow. It is the nature of OS, that when you create a border-less window it has no automatic capability of mouse-driven resizing. TotalTerminal's Visor window is forcibly turned to be borderless, to have its unique property:
https://github.com/binaryage/totalterminal/blob/master/totalterminal-plugin/TotalTerminal%2BVisor.mm#L288

**Comment by krackers** [comment]
If you set it as `NSResizableWindowMask` instead of `NSBorderlessWindowMask` you get resizing without the titlebar. Can do this via swizzling as follows:

```
@implementation myNSWindow
- (id) initWithContentRect:(NSRect)contentRect 
                          styleMask:(unsigned long)style 
                            backing:(unsigned long)backingStoreType 
                              defer:(BOOL)flag {
   return ZKOrig(id, contentRect, style == 0 ? 8 : style, backingStoreType, flag);
}
@end

```
